quasar.conf.js

```diff
@@ -109,7 +109,9 @@ module.exports = function (ctx) {
         CAPTCHA_NETWORK: process.env.CAPTCHA_NETWORK || 'telosTestnet',
         LOGIN_CONTRACT: process.env.LOGIN_CONTRACT,
         JOIN_CONTRACT: process.env.JOIN_CONTRACT,
-        JOIN_URI: process.env.JOIN_URI
+        JOIN_URI: process.env.JOIN_URI,
+        DOWNLOAD_WALLET_LINK_IOS: process.env.DOWNLOAD_WALLET_LINK_IOS,
```

Comment:
@evgenibir You would need to add these also to the github actions files ( in folder .github/workflow ) and would be good to set the default ones here like `DOWNLOAD_WALLET_LINK_IOS: 

process.env.DOWNLOAD_WALLET_LINK_IOS || 'http://itunes.apple.com/lb/app/id1659926348'` so that even without specifying the env variables deployments and local instances work.

Reply:
@Alex-IS fixed